src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -861,7 +861,7 @@ def _create_identity_instance(cli_ctx, *args, **kwargs):
     from .auth.identity import Identity
 
     # Only enable encryption for Windows (for now).
```

Comment:
This comment probably also needs to update to mention macOS now.

By the way, does this comment actually mean to say, "only allow fallback when running on Windows ..."?

Reply:
Oh. I forgot to update the comment!